src/runtime/types.ts

```diff
@@ -4,7 +4,8 @@ import type { ResvgRenderOptions } from '@resvg/resvg-js'
 import type { SatoriOptions } from 'satori'
 import type { AllowedComponentProps, Component, ComponentCustomProps, VNodeProps } from '@vue/runtime-core'
 import type { SharpOptions } from 'sharp'
-import type { NitroApp, WasmOptions } from 'nitropack'
+import type { NitroApp } from 'nitropack'
+import type { UnwasmPluginOptions as WasmOptions } from 'unwasm/plugin'
```

Comment:
Probably not related to this PR, but unwasm is **explicitly** a built-time tool. How `RuntimeCompatibilitySchema` would need to depend on it :(